zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -485,3 +488,31 @@ class TuyaSoilSensorVar02(CustomDevice):
             }
         },
     }
+
+
+class GiexBatteryStatus(t.enum8):
+    """Giex Soil Battery Status Enum."""
+
+    Low = 0x00
+    Middle = 0x01
+    High = 0x02
+
+
+(
+    TuyaQuirkBuilder("_TZE284_nhgdf6qr", "TS0601")  # Giex GX04
+    .tuya_battery(dp_id=15)
+    .tuya_temperature(dp_id=5, scale=10)
+    .tuya_soil_moisture(dp_id=3)
+    .tuya_enum(
+        dp_id=14,
+        attribute_name="battery_status",
+        enum_class=GiexBatteryStatus,
+        translation_key="battery_status",
+        fallback_name="Battery Status",
```

Comment:
Due to [Home Assistant naming](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/#:~:text=the%20rest%20of%20the%20words%20are%20lower%20case), this should be "Battery status".
```suggestion
        fallback_name="Battery status",
```